common/time/time.go

```diff
@@ -0,0 +1,20 @@
+// Package time provides functions to work with execution time
+package time
+
+import (
+	"fmt"
+	"regexp"
+	"runtime"
+	"time"
+)
+
+// Measure retrieves name of the caller function and prints it along with elapsed time
+// Typical call of Measure should be deffered in the beggining of the function to measure
+func Measure(start time.Time) {
+	elapsed := time.Since(start)
+	pc, _, _, _ := runtime.Caller(1)
+	pcFunc := runtime.FuncForPC(pc)
+	funcNameOnly := regexp.MustCompile(`^.*\.(.*)$`)
+	funcName := funcNameOnly.ReplaceAllString(pcFunc.Name(), "$1")
+	fmt.Printf("\n%s took %s\n", funcName, elapsed)
```

Comment:
I think it would be more useful for all-purpose if the func returns string.

Reply:
@levkoburburas did you mean to return string instead of printing to std out?

so, the caller will invoke it like `defer fmt.Printf(runtime.Measure(time.now()))` ?

Reply:
Yes 
* Since this is common package, we shouldn't assume that all will use fmt.Printf, for example in my project I would use
`defer log.Debug(runtime.Measure(time.now())`
* And for more readable code, it is preferable to show that this command prints something. Otherwise `Measure` doesn't give me any clue what does it really do

Reply:
@levkoburburas got it, sounds great